server.py

```diff
@@ -0,0 +1,55 @@
+import chat_proto.chat_pb2_grpc as chat_pb2_grpc
+import chat_proto.chat_pb2 as chat_pb2
+from concurrent import futures
+import grpc
+import time
+
+
+class ChatServiceServicer(chat_pb2_grpc.ChatServiceServicer):
+    
+
+    list_of_users = chat_pb2.Users()
+
+    list_of_messages = []
```

Comment:
Please move the vars that hold data out of the class. Data and logic should go separately.